apps/kyb-app/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>KYB - Collection Flow</title>
+    <script type="text/javascript" src="/config.js"></script>
```

Comment:
wrong path

Reply:
No in the dist index.html and config.js stay in the same directory